gravitee-am-identityprovider/gravitee-am-identityprovider-franceconnect/src/main/java/io/gravitee/am/identityprovider/franceconnect/authentication/FranceConnectAuthenticationProvider.java

```diff
@@ -43,6 +43,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
NPE on lines 110 and 151 because the query parameter "provider" is not used anymore.